django_gcloud_storage/__init__.py

```diff
@@ -241,4 +246,7 @@ def url(self, name):
         name = safe_join(self.bucket_subdir, name)
         name = prepare_name(name)
 
+        if self.use_unsigned_urls:
+          return "https://storage.googleapis.com/{}/{}".format(self.bucket.name, name)
```

Comment:
I noticed that I don't follow PEP8 myself fully (will fix that), so this doesn't _need_ to be changed, but the indentation of this line is off by two spaces